dts/bindings/video/nxp,imx-csi.yaml

```diff
@@ -14,7 +14,8 @@ properties:
   interrupts:
     required: true
 
-  sensor:
+  sdev:
```

Comment:
this name is not descriptive, also is this worth breaking backward compatibility

Reply:
@decsny : I can change to "_source_dev_" to be more descriptive. But the old name "sensor" is definitely not ok for the new support in RT11xx. As explained in the commit message, "sensor" is chosen because at that time, the CSI driver supports only RT10xx where it connects to a sensor. But in RT11xx, it connects to MIPI CSI2 instead.

Reply:
ok

Reply:
ok, done